core/storage/classifier/gae_models.py

```diff
@@ -140,17 +136,18 @@ def _generate_id(cls, exp_id):
     @classmethod
     def create(
             cls,
-            algorithm_id, # type: Text
-            interaction_id, # type: Text
-            exp_id, # type: Text
-            exp_version, # type: int
-            next_scheduled_check_time, # type: datetime.datetime
-            training_data, # type: Union[Dict[Text, Union[int, List[Text]]], List[Dict[Text, Union[int, List[Text]]]]]
-            state_name, # type: Text
-            status, # type: Text
-            algorithm_version # type: int
-    ):
-        # type: (...) -> Text
+            algorithm_id: str,
+            interaction_id: str,
+            exp_id: str,
+            exp_version: int,
+            next_scheduled_check_time: datetime.datetime,
+            training_data: Union[
+                Dict[str, Union[int, List[str]]],
+                List[Dict[str, Union[int, List[str]]]]],
```

Comment:
Should we separate this into a type variable?

Reply:
Done.